src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    className={classNames({
+      TodoInfo: true,
+      'TodoInfo--completed': todo.completed === true,
+    })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    <UserInfo user={todo.user} />
+  </article>
```

Comment:
```suggestion
  >
    <h2 className="TodoInfo__title">
      {todo.title}
    </h2>
    
    <UserInfo user={todo.user} />
  </article>
```
Such formatting helps to increase code readability